feat(Edge): Update loadbot warning (#1274)
maticnetwork/matic-docs: docs/edge/performance-reports/overview.md

  - loadbot
---

> :::caution 

Reviewer: ```suggestion
:::caution Polygon Edge loadbot is depreciated
```

> Please note that our `loadbot`, which was used for performing these tests, is now depreciated. 

Reviewer: ```suggestion
Please note that the `loadbot` used for performing these tests is depreciated.
```
